examples/pytorch/language-modeling/run_clm.py

```diff
@@ -52,7 +52,9 @@
 from transformers.trainer_utils import get_last_checkpoint
 from transformers.utils import check_min_version, send_example_telemetry
 from transformers.utils.versions import require_version
-
+# Initialize MLFlow
+import mlflow
+mlflow.set_tracking_uri(str(os.environ.get("TRACKING_URI")))
```

Comment:
LGTM. Should we set the tracking URL as env variable in the env.yml file?

Reply:
@HDThang Should the env var named `MLFLOW_TRACKING_URI`? 

Also, this line is not 100% fall-safe because of two reasons:
* I don't see `TRACKING_URI` explicitly set in any shell script
* If there is no such envvar, `os.environ.get` will return `None`, which may be unexpected  
I suggest adding a simple `if` check to this

Reply:
Oh, sorry to not mention this first. We should set the envvar TRACKING_URI before running the script